desk/app/groups.hoon

```diff
@@ -635,13 +637,15 @@
       ?+    rest.pole  ~
           [%can-read src=@ ~]
         =/  src  (slav %p src.rest.pole)
-        ?:  =(%open -.cordon.group)  `loob+!>(&)
```

Comment:
Note the removal of this check. Before, you could read everything in an open group, even secret channels.

Reply:
good catch